python/ql/test/library-tests/frameworks/requests/taint_test.py

```diff
@@ -0,0 +1,54 @@
+import requests
+
+from flask import Flask, request
+
+app = Flask(__name__)
+
+
+@app.route("/taint_test") # $ routeSetup="/taint_test"
+def test_taint(): # $ requestHandler
+    url = request.args['untrusted_input']
+
+    # response from a request to a user-controlled URL should be considered
+    # user-controlled as well.
+    resp = requests.get(url) # $ clientRequestUrlPart=url
+
+    requests.Response
+    requests.models.Response
+
+    ensure_tainted(
+        # see https://docs.python-requests.org/en/latest/api/#requests.Response
+        resp, # $ tainted
```

Comment:
```suggestion
    ensure_tainted(
        url, # $ tainted
        # see https://docs.python-requests.org/en/latest/api/#requests.Response
        resp, # $ tainted
```
I would add the url in here as a sanity check